src/popup/router/components/LinkButton.vue

```diff
@@ -13,8 +13,11 @@ export default {
     to: { type: String, required: true },
   },
   methods: {
-    handleClick() {
-      if (process.env.PLATFORM === 'cordova') openUrl(this.to);
+    handleClick(e) {
+      if (process.env.PLATFORM === 'cordova') {
+        e.preventDefault();
+        openUrl(this.to);
+      }
```

Comment:
Seems that the current version of Cordova opens external links fine without any additional stuff, proposing to merge #943 instead